pkg/terraform/exec/plugins/ibms3presign/resource_presign.go

```diff
@@ -0,0 +1,150 @@
+package ibms3presign
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/credentials"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/s3"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+var storageClass = []string{
+	"standard", "vault", "cold", "flex", "smart",
+}
+
+var endpointType = []string{
+	"public", "private", "direct",
+}
+
+func resourcePresign() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceIBMCOSBucketObjectPresignCreate,
+		Read:   resourceIBMCOSBucketObjectPresignCreate,
```

Comment:
i'm guessing pointing to the Create func for the Read value was intentional, as I don't see a Read func

Reply:
Yes, both are same.. this is intentional because of the no difference between them..